pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -757,6 +757,9 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 		// Run any post exec rules once all volume backup is triggered
 		driverCombo := a.checkVolumeDriverCombination(backup.Status.Volumes)
 		// If the driver combination of volumes are all non-kdmp, call the post exec rule immediately
+		// The checks below are for non-kdmp driver only, this is to ensure that snapshots for volumes
+		// are completed before calling the post-exec rules and ending the pre-exec pods if any.
+		// Additionally, it ensures we execute pos-exec rule only once in the lifetime of certain backup.
```

Comment:
Can you please add a sentence that why we are not have this for KDMP and mixed driver case?